lib/babe/babe.go

```diff
@@ -355,89 +355,109 @@ func (b *Service) initiate(epoch uint64) {
 }
 
 func (b *Service) invokeBlockAuthoring(epoch uint64) {
```

Comment:
does this service need to halt if `invokeBlockAuthoring` exits? If this function returned an `error`, appropriate actions could be taken in that case in `initiate`.

Reply:
yeah it should probably panic or `logger.Crit`. I can update that